go/signedexchange/README.md

```diff
@@ -1,3 +1,83 @@
-# gen-signedexchange
+# go/bundle
+This directory contains a reference implementation of [Signed HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html) spec.
 
-This directory will hold a reference implementation of a tool to sign exchanges, but for now, the version that best matches Chrome is in https://github.com/nyaxt/webpackage/tree/master/go/signedexchange .
+## Overview
+We currently provide two command-line tools: `gen-signedexchange` and `gen-certurl`.
+
+`gen-signedexchange` generates a signed exchange file. The `gen-signedexchange` command constructs HTTP request and response pair from given command line flags, attach a cryptographic signature of the pair, and serializes the result to output file.
+
+`gen-certurl` converts a X.509 certificate chain to `application/cert-chain+cbor` format, which is defined in the [Section 3.3 of the Signed HTTP Exchanges spec](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#rfc.section.3.3).
+
+You are also welcome to use the code as golang lib (e.g. `import "github.com/WICG/webpackage/go/signedexchange"`), but please be aware that the API is not yet stable and is subject to change any time.
+
+## Getting Started
+
+### Prerequisite
+golang environment needs to be set up in prior to using the tool. We are testing the tool on latest golang. Please refer to [Go Getting Started documentation](https://golang.org/doc/install) for the details.
```

Comment:
s/golang environment/The golang environment/
s/on latest golang/on the latest golang/ or "on the latest version of Go"
s/to [Go/to the [Go/
Maybe s/golang/Go/g: I don't think the Go docs ever talk about "golang".

Reply:
Done.